taca/illumina/HiSeqX_Runs.py

```diff
@@ -251,13 +260,19 @@ def look_for_lanes_with_10X_indicies(indexfile, ssparser):
     index_dict=parse_10X_indexes(indexfile)
     tenX_lanes = set() #Set to only get each lane once
     not_tenX_lanes = set()
+    tenX_samples = dict()
     for sample in ssparser.data:
         if sample['index'] in index_dict.keys():
             tenX_lanes.add(sample['Lane'])
+            sample_name = sample.get('Sample_Name') or sample.get('SampleName')
+            if tenX_samples.get(sample['Lane']):
```

Comment:
I think it would be easier to read if you use `if key in dict` instead of the `sample.get` method, so it would be `if sample['Lane'] in tenX_samples:`. And even nicer if you have a variable `lane` that you set before this if statement `lane=sample['Lane']`.